models/user/src/mc-user/mc-user-draft/generator.ts

```diff
@@ -0,0 +1,14 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { fakeBusinessRoles } from '../../constants';
+import type { TMcUserDraft } from '../types';
+
+const generator = Generator<TMcUserDraft>({
+  fields: {
+    email: fake((f) => f.internet.email()),
```

Comment:
Thank you for the updates!
I think `email` should be replaced with `password` https://github.com/commercetools/test-data/blob/448a17cdb0119faa43321bab6a0620fa0035a3c9/graphql-types/src/generated/mc.ts#L628-L633